src/Api/Controllers/UsersController.cs

```diff
@@ -8,8 +9,9 @@ namespace Api.Controllers;
 public class UsersController : ApiControllerBase
 {
     [HttpPost]
-    public async Task<ActionResult<UserDTO>> CreateUser(CreateUserCommand command)
+    public async Task<ActionResult<Result<UserDto>>> CreateUser(CreateUserCommand command)
     {
-        return await Mediator.Send(command);
+        var result = await Mediator.Send(command);
+        return Created("", Result<UserDto>.Succeed(result));
```

Comment:
Should use either Ok or pointing to an actual location

Reply:
isn't that create should return 201 ?

Reply:
leaving a blank in Created() means nothing, if you insist on using it then adhere to the full HTTP semantics by pointing to the real location that created resource lie in
If you cannot point to one, leave it as Ok is sufficient

Reply:
noted , thanks